ui/.dockerignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
+build/
+tmp/
```

Comment:
I think ```tmp```is not necessary here.

Reply:
We can add ```junit/``` to this list too

Reply:
Cypress created `tmp/cypress/screenshots/...` on my system.

Reply:
oh yes. it is for debug